components/brand/Testimonials.vue

```diff
@@ -288,4 +286,12 @@ onMounted(() => {
   transform: translateX(20px);
   opacity: 0;
 }
+.image-container{
+  @media (min-width: 1024px) {
+    height: 250px; 
+    width: 400px;
+  }
+    height: 170px; 
+    width: 320px;
```

Comment:
This is still not enough to fix the arrows covering the images. This is what it looks like on my Samsung S21 Ultra:

![image](https://github.com/Social-Currant/marketing-site/assets/61921720/74a76e87-51d5-4544-af4a-05da02417879)


One option to help improve this could be something like this:


Change from this

```scss
.arrow-container {
  top: 60%;

  @media (min-width: 625px) {
    top: 0%;
  }
}

.arrow-container.left {
  left: 8%;
}

.arrow-container.right {
  right: 8%;
}
```

to this

```scss
.arrow-container.left {
  left: 8%;
}

.arrow-container.right {
  right: 8%;
}

.arrow-container {
  top: 60%;

  @media (min-width: 625px) {
    top: 0%;
  }

  @media (max-width: 624px) {
    max-width: 32px;

    &.left {
      left: 4%;
    }

    &.right {
      right: 4%;
    }
  }
}
```

This way, the arrows will be more separated from each other and will also shrink and we could avoid the need to keep shrinking the images.

The expected result is this:

![image](https://github.com/Social-Currant/marketing-site/assets/61921720/188f7228-f075-4b31-a6b2-bb39c058f307)